packages-common/nanoflow-commons/package.json

```diff
@@ -5,13 +5,17 @@
   "scripts": {
     "build": "npm run build:tsc",
     "prebuild:tsc": "rm -rf dist/tsc",
-    "build:tsc": "..\"/../node_modules/.bin/tsc\" --project src/tsconfig.json",
-    "postbuild": "..\"/../node_modules/.bin/ts-node\" --project scripts/tsconfig.json ./scripts/copyActionsToTestProject.ts",
-    "lint": "..\"/../node_modules/.bin/eslint\" --config ../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/"
+    "build:tsc": "tsc --project src/tsconfig.json",
+    "postbuild": "ts-node --project scripts/tsconfig.json ./scripts/copyActionsToTestProject.ts",
+    "lint": "eslint --config ../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/"
   },
   "devDependencies": {
     "@babel/cli": "^7.6.0",
     "@babel/core": "^7.6.0",
-    "@types/node": "^12.0.2"
+    "@mendix/pluggable-widgets-tools": "8.9.0"
+  },
+  "dependencies": {
```

Comment:
Dont think we should have dependencies here, as this will be shipped with Studio Pro

Reply:
I am not sure. Yes, currently all dependency we have for js actions must be shipped with Studio Pro, but this is not how it will be forever. And if you take widgets, there you already can mix prebundled and not-prebundled dependencies. I don't think it will be nice to manually split dependencies to mark all pre-bundled as "dev".

Other modules don't align on this matter: `packages-native/actions` doesn't use this approach, while `packages-hybrid/hybrid-mobile-actions` does..

Reply:
Actually, now the majority of the packages were listing prebundled dependencies as dependencies and not as devDependencies.